test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Feeds/MultiSourcePackageMetadataProviderTests.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Moq;
+using NuGet.Packaging.Core;
+using NuGet.Protocol.Core.Types;
+using NuGet.Test.Utility;
+using NuGet.Versioning;
+using Xunit;
+
+namespace NuGet.PackageManagement.UI.Test.Feeds
+{
+    public class MultiSourcePackageMetadataProviderTests
+    {
+        [Fact]
+        public async Task MultiSourcePackageMetadataProvider_SingleRequestWhenGettingLatestPackageMetadata()
```

Comment:
Not sure how I feel about this convention as it does not align with the majority of the codebase but I'll change these tests.